README.md

```diff
@@ -30,11 +32,13 @@ with:
     username: ${{ secrets.AQUA_USER }}
     password: ${{ secrets.AQUA_PASSWORD }}
     ## URL for aquasec CSP
-    url: https://aquasec.example.com
+    url: https://12345.cloud.aquasec.com
```

Comment:
12345?

Reply:
It's an example - it can be anything.

Reply:
Probably should add a comment "Copy the URL from your Aquasec Image Scanning environment, in the cloud it looks like 12345.cloud.aquasec.com"